packages/sit-onyx/src/styles/grid.ct.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable playwright/expect-expect */
 import type { Locator, Page } from "@playwright/test";
+import { ONYX_BREAKPOINTS, type OnyxBreakpoint } from "@sit-onyx/shared/breakpoints";
```

Comment:
Why is there a dedicated `/breakpoints` path?
Shouldn't we just export all relevant exports directly from `@sit-onyx/shared`?

Reply:
The two exports to be bundled together, as they are address different run targets

Reply:
Should we then instead use a `@sit-onyx/shared/node` or `@sit-onyx/shared/browser` path so we don't have to create a new path in the future for every new feature? With the current name its limited to only include breakpoints

Reply:
Ideally, yes. But looking at the PR, its age and scope this is too much effort for now. Fo-Up was created: [#2056](https://github.com/SchwarzIT/onyx/issues/2056)